src/Test/L1/Worker/WorkerL1Tests.cs

```diff
@@ -199,15 +200,19 @@ public async Task StepTarget_RestrictedMode()
         // TODO: When NuGet works cross-platform, remove these traits. Also, package NuGet with the Agent.
         [Trait("SkipOn", "darwin")]
         [Trait("SkipOn", "linux")]
-        public async Task SignatureEnforcementMode_PassesWhenAllTasksAreSigned()
+        public async Task SignatureVerification_PassesWhenAllTasksAreSigned()
         {
             try
             {
                 // Arrange
                 SetupL1();
                 FakeConfigurationStore fakeConfigurationStore = GetMockedService<FakeConfigurationStore>();
                 AgentSettings settings = fakeConfigurationStore.GetSettings();
-                settings.Fingerprint = _fingerprint;
+                settings.SignatureVerification = new SignatureVerificationSettings()
```

Comment:
I think we also want a test that passes, in error mode, when no fingerprints are provided. This will be the new default mode.

Reply:
Good point. I'll change both the success and failure Facts into Theories that handle both the fingerprint and no fingerprint variants.